doc/releases/release-notes-3.7.rst

```diff
@@ -745,6 +745,20 @@ Drivers and Sensors
   * Added release API
   * Added support for mode selection via the device tree
 
+* MSPI
+
+  * Add the new :ref:`MSPI(Multi-bit SPI) <mspi_api>` API, enabling support for advanced
+    SPI controllers and peripherals that typically require command, address and data phases.
+    The API now supports single wire SDR up to hex wires DDR communication in sync/async ways.
+  * Added APS6404L QPI pSRAM device driver that uses the MSPI API.
```

Comment:
```suggestion
  * Added APS6404L QPI pSRAM device driver.
```
Nit: I would probably just remove all 3 "that uses the MSPI API" from these 3 lines. It is implied from being in the list.